src/main/resources/application.properties

```diff
@@ -42,7 +42,6 @@ management.health.diskspace.enabled=true
 # Enable the metrics endpoint
 management.endpoint.metrics.enabled=true
 
-azure.app_insights_key=${AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY:false}
```

Comment:
This looks like wrong key anyway, but better to not have a dummy default in this case.